code-challenges.test.js

```diff
@@ -24,10 +24,35 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+//PROCESS:
+    //describe the name of the function - fibArray
+    //it method that describes what the function does - takes in a number greater than 2 and returns an array of that length containing the numbers of the Fibonacci.
+    //expect the input of 6 to output: [1, 1, 2, 3, 5, 8] and the input of 10 to output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    var num1 = 6
+    var num2 = 10
 
+    describe("fibArray", () => {
+        it("takes a number greater than 2 and returns an array of that length containing the numbers of the Fibonacci", () => {
+            expect(fibArray(num1)).toEqual([1, 1, 2, 3, 5, 8])
+            expect(fibArray(num2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+        })
+    })
 
 // b) Create the function that makes the test pass.
+//PROCESS:
+    //declare a function named fibArray that takes in a number greater than 2 and returns an array of length of number of the Fibonacci.
+    //create a for loop that will iterate at the length of the number 
+    //push into empty array.. ?
 
+    const fibArray = (number) {
+       let fibbed = []
+        for(let i=2; i<number; i++){
+            fibbed.push()
+        }
+    }
+
+//I saved this one for last and couldnt figure it out on time. :/
```

Comment:
No worries, You can work on this stuff over the weekend if you like!  Do you mind sharing your pseudo code of the problem with me? Feel free to post it here as a reply to the comment--

Reply:
After reviewing the assessment as a class, I learned there is a code for the fibonacci sequence which is "x = x-1 + x-2"
Pseudocode for coding-challenge 1:
1. Create a funciton- fibArrray 
2. parameter- number
3. declare a variable equal to the array I will push into - let answerArrray = [1, 1] (start at [1,1] since we only care for numbers greater than 2)
 4. create a for loop (let i=2; i<number; i++)
 5. push into answerArray and use fibonacci code- (answerArray[i-1] + answerArray[i-2]
 6. return answerArray